samples/Mvc.Client/Mvc.Client.csproj

```diff
@@ -17,6 +17,61 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.Twitter" Version="$(TwitterProviderVersion)" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
These aren't needed, line 11 adds all the projects automatically.

Reply:
I know it's a vs for mac. I remove this before next commit